diff --git a/src/cmd/ksh93/tests/builtins.sh b/src/cmd/ksh93/tests/builtins.sh
index 55b1f4f2..c2caa790 100644
--- a/src/cmd/ksh93/tests/builtins.sh
+++ b/src/cmd/ksh93/tests/builtins.sh
@@ -506,6 +506,8 @@ then
     rmdir "$pwd/f1"
 fi

+log_info "HOME=|$HOME| USER=|$USER|"
+
 # =======
 # Verify `~` expansion works if `$HOME` is not set.
 # Regression test for https://github.com/att/ast/issues/1391
