Index: ksh-93u+20120801/src/lib/libast/features/stdio
===================================================================
--- ksh-93u+20120801.orig/src/lib/libast/features/stdio
+++ ksh-93u+20120801/src/lib/libast/features/stdio
@@ -6,6 +6,8 @@ cat{
 	#define _FILE_DEFINED	1
 	#define _FILE_defined	1
 	#define _FILEDEFED	1
+	#define __FILE_defined	1
+	#define ____FILE_defined	1
 
 	#ifndef __FILE_TAG
 	#define __FILE_TAG	_sfio_s
Index: ksh-93u+20120801/src/lib/libast/include/ast.h
===================================================================
--- ksh-93u+20120801.orig/src/lib/libast/include/ast.h
+++ ksh-93u+20120801/src/lib/libast/include/ast.h
@@ -58,9 +58,21 @@ struct _sfio_s;
 #ifndef	__FILE_typedef
 #define __FILE_typedef	1
 #endif
+#ifndef _FILE_DEFINED
+#define _FILE_DEFINED   1
+#endif
+#ifndef   _FILE_defined
+#define _FILE_defined   1
+#endif
 #ifndef _FILEDEFED
 #define _FILEDEFED	1
 #endif
+#ifndef __FILE_defined
+#define __FILE_defined  1
+#endif
+#ifndef ____FILE_defined
+#define ____FILE_defined  1
+#endif
 #endif
 
 /*
