Description: Fix typos and lintian warnings
Author: Anuradha Weeraman <aweeraman@gmail.com>
Forwarded: no
Last-Update: 2020-01-18

--- a/src/cmd/ksh93/RELEASE93
+++ b/src/cmd/ksh93/RELEASE93
@@ -313,7 +313,7 @@ of ksh.
 	    the value of a name reference contained a positional
 	    parameter and the name reference was not defined inside
 	    a function has been fixed.
-	o.  Arithmetic expressions now correctly handle hexidecimal
+	o.  Arithmetic expressions now correctly handle hexadecimal
 	    constants.
 	p.  A bug in which integer variables could be expanded
 	    with a leading 10# when declared with typeset -i
--- a/src/cmd/ksh93/data/builtins.c
+++ b/src/cmd/ksh93/data/builtins.c
@@ -968,12 +968,12 @@ USAGE_LICENSE
 "[+?If \afirst\a is specified and \alast\a is omitted, then \alast\a will "
 	"default to \afirst\a unless \b-l\b is specified in which case "
 	"it will default to \b-1\b.]"
-"[+?If no editor is specified, then the editor specfied by the \bHISTEDIT\b "
+"[+?If no editor is specified, then the editor specified by the \bHISTEDIT\b "
 	"variable will be used if set, or the \bFCEDIT\b variable will be "
 	"used if set, otherwise, \bed\b will be used.]"
 "[e]:[editor?\aeditor\a specifies the editor to use to edit the history "
 	"command.   A value of \b-\b for \aeditor\a is equivalent to "
-	"specifiying the \b-s\b option.]"
+	"specifying the \b-s\b option.]"
 "[l?List the commands rather than editing and reexecuting them.]"
 "[N]#[num?Start at \anum\a commands back.]" 
 "[n?Suppress the command numbers when the commands are listed.]"
@@ -1196,9 +1196,9 @@ USAGE_LICENSE
 	"[+-?The escape sequence \b\\C[.\b\aname\a\b.]]\b expands to "
 		"the collating element \aname\a.]"
 	"[+-?The escape sequence \b\\x{\b\ahex\a\b}\b expands to the "
-		"character corresponding to the hexidecimal value \ahex\a.]"
+		"character corresponding to the hexadecimal value \ahex\a.]"
 	"[+-?The escape sequence \b\\u{\b\ahex\a\b}\b expands to the unicode "
-		"character corresponding to the hexidecimal value \ahex\a.]"
+		"character corresponding to the hexadecimal value \ahex\a.]"
 	"[+-?The format modifier flag \b=\b can be used to center a field to "
 		"a specified width.]"
 	"[+-?The format modifier flag \bL\b can be used with the \bc\b and "
@@ -1654,7 +1654,7 @@ USAGE_LICENSE
 "[l?Without \b-i\b, sets character mapping to \btolower\b. When used "
 	"with \b-i\b, \b-E\b, or \b-F\b indicates long variant.]"
 "[m?Move.  The value is the name of a variable whose value will be "
-	"moved to \aname\a.  The orignal variable will be unset.  Cannot be "
+	"moved to \aname\a.  The original variable will be unset.  Cannot be "
 	"used with any other options.]"
 "[n?Name reference.  The value is the name of a variable that \aname\a "
 	"references.  \aname\a cannot contain a \b.\b.  Cannot be use with "
--- a/src/lib/libcmd/chmod.c
+++ b/src/lib/libcmd/chmod.c
@@ -36,7 +36,7 @@ USAGE_LICENSE
 	"of changes to make, or an octal number representing the bit "
 	"pattern for the new permissions.]"
 "[+?Symbolic mode strings consist of one or more comma separated list "
-	"of operations that can be perfomed on the mode. Each operation is of "
+	"of operations that can be performed on the mode. Each operation is of "
 	"the form \auser\a \aop\a \aperm\a where \auser\a is zero or more of "
 	"the following letters:]{"
 	"[+u?User permission bits.]"
--- a/src/lib/libcmd/uname.c
+++ b/src/lib/libcmd/uname.c
@@ -35,7 +35,7 @@ USAGE_LICENSE
 "	standard output. When options are specified, one or more"
 "	system characteristics are written to standard output, space"
 "	separated, on a single line. When more than one option is specified"
-"	the output is in the order specfied by the \b-A\b option below."
+"	the output is in the order specified by the \b-A\b option below."
 "	Unsupported option values are listed as \a[option]]\a. If any unknown"
 "	options are specified then the local \b/usr/bin/uname\b is called.]"
 "[+?If any \aname\a operands are specified then the \bsysinfo\b(2) values"
