Description: Use TMPDIR in bin/package
Origin: https://cvsweb.openbsd.org/cgi-bin/cvsweb/ports/shells/ksh93/patches/patch-bin_package?rev=1.1.1.1&content-type=text/x-cvsweb-markup
Forwarded: not-needed
Last-Update: 2021-08-22

--- a/bin/package
+++ b/bin/package
@@ -2823,7 +2823,7 @@ cat $INITROOT/$i.sh
 			# check if $CC (full path $cc) is a cross compiler
 
 			(
-				cd /tmp || exit 3
+				cd ${TMPDIR:-/tmp} || exit 3
 				cp $INITROOT/hello.c pkg$$.c || exit 3
 				$cc -o pkg$$.exe pkg$$.c > pkg$$.e 2>&1 || {
 					if $cc -Dnew=old -o pkg$$.exe pkg$$.c > /dev/null 2>&1
